backend/src/main/resources/application-dashboards-staging.yaml

```diff
@@ -2,7 +2,7 @@ dashboards:
   organisms:
     covid:
       lapis:
-        url: "https://lapis.cov-spectrum.org/open/v2"
+        url: "https://gs-staging-1.int.genspectrum.org/open/v2"
```

Comment:
do we have covid data on genspectrum? I don't see it in the `organismValues.yaml`?

Reply:
We don't have it in our Loculus. The LAPIS instances run separately outside of Loculus.